xml/depl_maintenance.xml

```diff
@@ -1503,6 +1503,295 @@ migrate</screen>
+       </para>
+      </question>
+      <answer>
+       <para>
+	It could happen that it takes the node too long to come up, which cause
+	a timeout. Check the node manually, and make sure it is online. Repeat then the step.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	N number of nodes were provided to compute upgrade using
+	<command>crowbarctl upgrade nodes node_1,node_2,...,node_N</command>,
+	but less then N were actually upgraded.
+       </para>
+      </question>
+      <answer>
+       <para>
+	If the live migration cannot be performed for certain nodes due to a timeout,
+	&crow; upgrades only the nodes that it was able to
+	live-evacuate. Since some nodes have been upgraded, it is possible that
+	more resources will available for live-migration if you try to run this
+	step again. See also question on live migration failure.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	Node has failed at the initial chef client run stage.
+       </para>
+      </question>
+      <answer>
+       <para>
+	An unsupported entry in the configuration file may prevent a service
+	from starting. This, in turn, causes the node to fail at the initial
+	chef client run stage. Checking the
+	<filename>/var/log/crowbar/crowbar_join/chef.*</filename> log files on
+	the node is a good starting point.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	I need to change &ostack-bare; configuration during the upgrade but I cannot access &crow;.
+       </para>
+      </question>
+      <answer>
+       <para>
+	&crow; &wi; is accessible only when upgrade is completed or
+	when it is postponed. Postponing the upgrade can be done only after
+	upgrading all &contrnode;s using the <command>crowbarctl upgrade nodes
+	postpone</command> command. You can then access &crow; and
+	even save some modification. It is then necessary to resume the upgrade
+	using the <command>crowbarctl upgrade nodes resume</command> command
+	before you can continue with the upgrade of rest of the nodes.
+       </para>
+      </answer>
+     </qandaentry>
+     <qandaentry>
+      <question>
+       <para>
+	Failure occurred when evacuating routers.
+       </para>
+      </question>
+      <answer>
+       <para>
+	Check the <filename>/var/log/crowbar/node-upgrade.log</filename> file on
+	the node that performs the router evacuation (it should be mentioned in
+	the error message). The ID of the router that failed to migrate (or the
+	affected network port) is logged to
+	<filename>/var/log/crowbar/node-upgrade.log</filename>). Use the
+	&ostack-bare; CLI tools to check the state of the affected router and
+	its ports. Fix manually, if necessary. This can be done, for example,
+	by bringing the router or port up and down again. The following
+	commands can be useful for solving the issue: <command>openstack router
```

Comment:
Could you format the commands so they is only one on a line? (Just like with nova-migrations above)